modules/sandboxed-containers-triggering-installation-kata-runtime.adoc

```diff
@@ -3,5 +3,67 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-triggering-installation-kata-runtime_{context}"]
-
 = Triggering the installation of the Kata runtime
+
+You must create one `KataConfig` custom resource (CR) to trigger the {sandboxed-containers-operator} to do the following:
+
+* Install the needed {op-system} extensions, such as QEMU and `kata-containers`, on your {op-system} node.
+* Ensure that the runtime, link:https://github.com/cri-o/cri-o[CRI-O], is configured with the correct Kata runtime handlers.
+* Create `RuntimeClass` with necessary configurations for additional overhead caused by virtualization and the required additional processes.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Create the `KataConfig` resource:
```

Comment:
In the command-line version, we explain how to `oc apply -f` the file. Is it customary to explain the web-based procedure the same way?

Reply:
@c3d This is the content that we have here. Should the code change?

> Create the `KataConfig` resource:

` apiVersion: kataconfiguration.openshift.io/v1`

`  kind: KataConfig`

  `metadata:`

   `name: cluster-kataconfig`

Reply:
Agree. We need a `oc apply -f <file>.yaml` or `oc create -f <file>.yaml`